### PR TITLE
Add documentation on docker use

### PR DESCRIPTION
**Pull request by sapiderman**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  

1. Closing #486 
2. Add more details on docker use

## How did you implement / how did you fix it  
1.  Add docs 
2. Add examples, some taken from [discussions/485](https://github.com/hyperjumptech/monika/discussions/485)
3. Add link to docker documentation on their website

### CODE REVIEW

File: docs/src/pages/tutorial/run-in-docker.md [hunk 1]
```diff
@@ -3,17 +3,24 @@ id: run-monika-in-docker
...
+
+```bash
+docker pull hyperjump/monika
+```
+
+Once you've pulled the latest image, pass it your monika yaml configuration to the container. From the same directory, you can run:
 
 ```bash
 # Run Monika in foreground
 docker run --name monika -v ${PWD}/monika.yml:/config/monika.yml --detach hyperjump/monika:latest
```
Comment (samsunoor) [inline docs/src/pages/tutorial/run-in-docker.md:16]: this command with `--detach` flag is conflicting with the `foreground` in the comment above mas

File: docs/src/pages/tutorial/run-in-docker.md [hunk 2]
```diff
@@ -25,3 +32,32 @@ Or you can stop the container with
...
+
+```bash
+docker run --name monika_interactive \
+    -v ${PWD}/myConfig.yml:/config/monika.yml \
+    -d hyperjump/monika:latest monika --help
+```
+
+For further docker commands and documentation, visit the official Docker [documentation here](https://docs.docker.com/engine/reference/commandline/run/).
+
+If all else fails, hit us up at https://github.com/hyperjumptech/monika.
```
Comment (samsunoor) [inline docs/src/pages/tutorial/run-in-docker.md:63]: maybe put direct link to [monika discussions](https://github.com/hyperjumptech/monika/discussions) or [file a bug report](https://github.com/hyperjumptech/monika/issues)?

**Review by samsunoor — APPROVED** [review]

File: docs/src/pages/tutorial/run-in-docker.md (hunk 1 above)
Comment (sapiderman) [reply docs/src/pages/tutorial/run-in-docker.md]: oops thanks. copas. fixed

File: docs/src/pages/tutorial/run-in-docker.md (hunk 2 above)
Comment (sapiderman) [reply docs/src/pages/tutorial/run-in-docker.md:63]: good tip. thanks.

**Review by haricnugraha — APPROVED** [review]

File: docs/src/pages/tutorial/run-in-docker.md [hunk 3]
```diff
@@ -3,17 +3,24 @@ id: run-monika-in-docker
 title: Run Monika in Docker
 ---
 
-Monika is available as a docker image. Paste a copy of your configuration file `monika.yml` into the current directory.
-From the same directory, you can run:
+Monika is available as a docker image. You can find the image in the docker hub as `hyperjump/monika`, or pull the image with the following command:
+
+```bash
+docker pull hyperjump/monika
```
[the remark this answers is not in the document]
Comment (sapiderman) [reply docs/src/pages/tutorial/run-in-docker.md:9]: True. The point there is to identify the repository/image really so they can make their own container.

**Pull request merged**